components/aws/sagemaker/tests/integration_tests/component_tests/test_workteam_component.py

```diff
@@ -7,28 +7,15 @@
 
 
 def create_workteamjob(
-    kfp_client, experiment_id, region, sagemaker_client, test_file_dir, download_dir
+    kfp_client, test_params, experiment_id, region, sagemaker_client, download_dir
 ):
```

Comment:
I think this change might break groundtruth test because this method is a dependency for it, it is not creating workteam via its pipeline, rather using this method